Dockerfile

```diff
@@ -62,6 +62,7 @@ RUN set -ex \
     && pip install pyasn1 \
     && pip install apache-airflow[crypto,celery,postgres,hive,jdbc,mysql]==$AIRFLOW_VERSION \
     && pip install 'celery[redis]>=4.1.1,<4.2.0' \
+    && pip install paramiko \
```

Comment:
Airflow includes an `extras` group for ssh, which includes a few more dependencies. It might make more sense to install `apache-airflow[ssh]` than just `paramiko`.

Reply:
Thanks for pointing this out!
I changed this PR to use the existing group for ssh instead.